src/app/home-page/widget/requests/requests.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="px-4 py-2">
+  <h3>Requests</h3>
+  <div *ngIf="myOrganizationInvites?.length == 0 && myOrganizationRequests?.length == 0; else hasRequests">
+    <p>You have no requests</p>
+  </div>
+
+  <ng-template #hasRequests>
+    <p *ngIf="myOrganizationInvites?.length > 0">
+      {{ myOrganizationInvites?.length }} organization invite<span *ngIf="myOrganizationInvites?.length != 1">s</span>
```

Comment:
Tiny nit, I would say `> 1`, because you've already established that it's more than zero.

Reply:
In this case we don't know this, we just know that at least one of the variables in non-empty, so we cannot say that we know that this variable is at least 1.

Reply:
I think the names might be a bit confusing, they are too similar.